dolphinscheduler-task-plugin/dolphinscheduler-task-datax/src/main/java/org/apache/dolphinscheduler/plugin/task/datax/DataxParameters.java

```diff
@@ -206,6 +211,14 @@ public void setJobSpeedRecord(int jobSpeedRecord) {
         this.jobSpeedRecord = jobSpeedRecord;
     }
 
+    public int getChannelCount() {
+        return channelCount;
+    }
+
+    public void setChannelCount(int channelCount) {
+        this.channelCount = channelCount;
+    }
+
```

Comment:
Please use `@Data` to generate getter and setter.

Reply:
Done.